dune-project

```diff
@@ -23,13 +23,11 @@
   sexplib
   ppx_deriving
   ppx_sexp_conv
-  sha
+  (sha (>= 1.15.1))
   sqlite3
   (obuilder-spec (= :version))
-  (ocaml (>= 4.10.0))
-  (alcotest-lwt :with-test))
- (conflicts
-  (result (< "1.5"))))
+  (ocaml (>= 4.14.0))
```

Comment:
This is jumping too far forward, 4.12 released on 2021-02-24 would be my suggested lower bound.

Reply:
I got fixes for Windows into 4.13 and 4.14. Maybe OBuilder only uses the fixes I've pushed in Lwt, but now that I know that the two libraries have the same behavior on what's relevant to me, I'd rather not think of the possible discrepancies between OCaml 4.12 and Lwt 5.6. Besides iirc I'm using `Unix.realpath` from 4.13 in the Windows branch. And I've only ever tested with 4.14.